docs/languages/javascript.md

```diff
@@ -223,48 +223,16 @@ The  `editor.codeActionsOnSave` setting lets you configure a set of Code Actions
     "source.organizeImports": true,
 }
 ```
-You can also enable or disable which Code Actions are run on save per language using a language specific setting:
-```json
-"[javascript]": {
-    "editor.codeActionsOnSave": {
-        "source.organizeImports": true
-    }
-}
-```
+
 Here are some source actions:
 
 * `"organizeImports"` -  Enables organize imports on save.
 * `"fixAll"` - Auto Fix on Save computes all possible fixes in one round (for all providers including ESLint).
 * `"fixAll.eslint"` -  Auto Fix only for ESLint.
 * `"addMissingImports"` - Adds all missing imports on save.
 
-The TSLint extension already uses `source.fixAll` Code Action kind to implement fix all and auto fix on save.
+See [Node.js/JavaScript](/docs/nodejs/working-with-javascript) for more information.
```

Comment:
I don't know if you prefer to link to a subsection instead but I found the subsection links just in case:

https://code.visualstudio.com/docs/getstarted/settings#_language-specific-editor-settings

For organizeImports specifically in Node.js/JavaScript:
https://code.visualstudio.com/docs/languages/javascript#_organize-imports